### ISSUE TITLE
I don't see any improvements after installing 2dfx for GTA SA...

### ISSUE THREAD

**Issue by Ninja92X** [ThirteenAG/III.VC.SA.IV.Project2DFX#19]
My title is very clear on this. As stated in the text file of 2dfx, on how to install it. I placed the files in the game directory, but unfortunately for me, it had no effect on the game, so i'll hope that somebody could help me with this issue.

**Comment by spartaque12** [comment]
did u intall asi loader ? also are u usin gta sa 1.0?

**Comment by Ninja92X** [comment]
I have installed asi loader, and i use  GTA  SA 1.0
The version on my pc, was originally  version 1.01. but i have downgraded
it to 1.0
What the thing is with GTA SA, is that the effect of 2dfx is not so much
visble, as it is on GTA VC & III.
I had also made a change in the drawdistance of the game, by replacing the
original file with a one the double's it 4 times. And even when i placed
the original back, the effect of 2dfx was verry weak on GTA SA.
So i was wondering how to fix this.


2017-10-28 20:25 GMT+02:00 Roman <notifications@github.com>:
